src/xcode/ENA/ENAUITests/ExposureDetection/ENAUITests_04b_ExposureDetection.swift

```diff
@@ -33,6 +33,30 @@ class ENAUITests_04b_ExposureDetection: CWATestCase {
 		app.buttons[AccessibilityIdentifiers.SurveyConsent.acceptButton].waitAndTap()
 	}
 
+	func test_navigateToInformationGuideHygiene() throws {
```

Comment:
Isn't there a change in UI test for the current screens? Also I can not see any screenshot.

Reply:
no screenshot are requested in the ticket